*** CgProto.ts	2022-07-09 12:25:54.000000000 -0700
--- CgProto-patched.ts	2022-07-09 12:25:54.000000000 -0700
*************** export class CgMessage extends pb_1.Mess
*** 303,309 ****
      serializeBinary(): Uint8Array {
          return this.serialize();
      }
!     static deserializeBinary(bytes: Uint8Array): CgMessage {
          return CgMessage.deserialize(bytes);
      }
  }
--- 303,309 ----
      serializeBinary(): Uint8Array {
          return this.serialize();
      }
!     static override deserializeBinary(bytes: Uint8Array): CgMessage {
          return CgMessage.deserialize(bytes);
      }
  }
