*** CgProto.ts	2022-07-09 15:09:39.000000000 -0700
--- CgProto-patched.ts	2022-07-09 15:09:38.000000000 -0700
*************** export class CgMessage extends pb_1.Mess
*** 304,310 ****
      serializeBinary(): Uint8Array {
          return this.serialize();
      }
!     static deserializeBinary(bytes: Uint8Array): CgMessage {
          return CgMessage.deserialize(bytes);
      }
  }
--- 304,310 ----
      serializeBinary(): Uint8Array {
          return this.serialize();
      }
!     static override deserializeBinary(bytes: Uint8Array): CgMessage {
          return CgMessage.deserialize(bytes);
      }
  }
