*** GgProto.ts	2022-07-10 14:56:40.000000000 -0700
--- GgProto-patched.ts	2022-07-10 14:56:39.000000000 -0700
*************** export class Rost extends pb_1.Message {
*** 120,126 ****
      serializeBinary(): Uint8Array {
          return this.serialize();
      }
!     static deserializeBinary(bytes: Uint8Array): Rost {
          return Rost.deserialize(bytes);
      }
  }
--- 120,126 ----
      serializeBinary(): Uint8Array {
          return this.serialize();
      }
!     static override deserializeBinary(bytes: Uint8Array): Rost {
          return Rost.deserialize(bytes);
      }
  }
*************** export class GgMsgBase extends pb_1.Mess
*** 348,354 ****
      serializeBinary(): Uint8Array {
          return this.serialize();
      }
!     static deserializeBinary(bytes: Uint8Array): GgMsgBase {
          return GgMsgBase.deserialize(bytes);
      }
  }
--- 348,354 ----
      serializeBinary(): Uint8Array {
          return this.serialize();
      }
!     static override deserializeBinary(bytes: Uint8Array): GgMsgBase {
          return GgMsgBase.deserialize(bytes);
      }
  }
